lib/widgets/bottom_sheets/chat_piece_jointe_bottom_sheet.dart

```diff
@@ -73,6 +118,7 @@ class _SelectPictureButton extends StatelessWidget {
       onTap: () async {
         final result = await ImagePickerWrapper.pickSingleImage();
         if (result != null) {
+          // Navigator.pop is called in a future to avoir error with context:
```

Comment:
to avoiD

Reply:
Finalement j'ai retiré les futures et fait des context.mounted